### PR TITLE
[Documentation][Cookbook] How to embed a list of products?

### PR DESCRIPTION
**Pull request by TheMadeleine**
| Q | A |
| --- | --- |
| Doc fix? | no |
| New docs? | yes |
| BC breaks? | no |
| Related tickets | partially #5275 |
| License | MIT |

### CODE REVIEW

File: docs/cookbook/embedding-products.rst [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+                ->andWhere('o.enabled = true')
+                ->andWhere('channel = :channel')
+                ->innerJoin('o.taxons', 'taxon')
+                ->andWhere('taxon.code = :code')
+                ->setParameter('channel', $channel)
+                ->setParameter('code', $code)
+                ->setMaxResults($count)
+                ->getQuery()
+                ->getResult()
+                ;
```
Comment (NeverResponse) [inline docs/cookbook/embedding-products.rst]: 👈  the `;`

File: docs/cookbook/embedding-products.rst [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+------------------------------------------------
+
+Having a new path, you can call it in a twig template. Remember that you need to have your **taxon as a variable available there**.
+Render the list using a simple template first.
+
+.. code-block:: twig
+
+    {{ render(url('app_shop_partial_product_index_latest_by_taxon_code', {'code': taxon.code, 'count': 5, 'template': '@SyliusShop/Product/_simpleList.html.twig'})) }}
+
+Done. In the taxon view where you have rendered the new url you will see a list of 5 newest products as a simple list.
```
Comment (NeverResponse) [inline docs/cookbook/embedding-products.rst]: `you will see a simple list of/containing 5 newest products.` also you might add ` of/from this taxon.` as well, tho i think it's a bit overkill

File: docs/cookbook/embedding-products.rst [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+Create a new method for the product repository
+----------------------------------------------
+
+To cover the usecase we have imagined we will need a new method on the product repository: ``findLatestByChannelAndTaxonCode()``.
+
+.. tip::
+
+    First learn how to customize repositories in :doc:`the customization docs here </customization/repository>`.
+
+The new repository method will take a channel object(retrieved form channel context), a taxon code and the count of items that you want to find.
```
Comment (NeverResponse) [inline docs/cookbook/embedding-products.rst]: Missing a space after `channel object`, `form` => `from`

**Comment by pjedrzejewski** [comment]
Nice! Thank you Magda! :)

**Pull request merged**